src/components/Header.tsx

```diff
@@ -84,7 +84,9 @@ export function Header({ user }: UserProps) {
 												<div>
 													<Menu.Button className='focus:underline-2 focus:underline-none flex rounded-full bg-gray-800 text-sm'>
 														<span className='sr-only'>Open user menu</span>
-														<div className='text-base font-medium text-gray-300 hover:text-white'>Tom Cook</div>
+														<div className='text-base font-medium capitalize text-gray-300 hover:text-white'>
+															{user?.email?.slice(0, user.email?.indexOf('@'))}
```

Comment:
suggestion (non-blocking): we use the email field here and on L105 and L206, I'd consider extracting somewhere on the top of the component:
```
const { email } = user;
const [username] = email.split('@');
```
And then use those values.
This way there are less duplicates of logic.

Reply:
I don't think we can do `const { email } = user;` as `user` is undefined at first before the login session is fetched.

But `user?.email?.slice(0, user.email?.indexOf('@'))` can be simplified to `user?.email?.split('@')[0]` for now. Thanks for the idea :)

Reply:
It's very hard to read on a mobile device. I see what you mean, there's a check for user which I haven't seen the first time. I think it could be improved overall, but it's none of concerns for this particular PR. Will approve as soon as you are ready.

Reply:
Done. Thanks for the feedback.